app/view/grid/Grid.js

```diff
@@ -110,7 +110,14 @@ Ext.define('CpsiMapview.view.grid.Grid', {
             text: 'Clear Sorting',
             glyph: 'f039@FontAwesome',
             handler: 'onClearSort'
-        }, '->',
+        },
+        {
+            xtype: 'button',
```

Comment:
Have this button hidden by default, but display for the ExampleGrid? Otherwise all existing grids that inherit from this class get the button.

Reply:
Good point. I changed it so the button will only shown if the layer has the property "gridFilter". I will push the commits later

Reply:
here is the respective commit c8e8c23
@geographika please let me know if we can close this conversation